bluechip/chain.json

```diff
@@ -0,0 +1,74 @@
+{
+    "$schema": "../chain.schema.json",
+    "chain_name": "bluechip",
+    "status": "live",
+    "website": "https://www.bluechip.link",
+    "network_type": "mainnet",
+    "pretty_name": "BlueChip",
+    "chain_id": "bluechip_1",
+    "bech32_prefix": "bcp",
+    "daemon_name": "bluechipd",
+    "node_home": "$HOME/.bluechip",
+    "slip44": 118,
+    "key_algos": [
+        "secp256k1"
+      ],
+    "fees": {
+      "fee_tokens": [
+        {
+          "denom": "ubluechip",
+          "fixed_min_gas_price": 0,
+          "low_gas_price": 0,
+          "average_gas_price": 0.025,
+          "high_gas_price": 0.04
+        }
+      ]
+    },
+    "staking": {
+      "staking_tokens": [
+        {
+          "denom": "ubluechip"
+        }
+      ]
+    },
+    "description": "Stake to the BlueChip chain, interact with the creator pools, and subscribe through the creator subscription contracts.",
+    "codebase": {
+      "git_repo": "https://github.com/Bluechip23/bluechip/tree/main",
+      "recommended_version": "v1",
+      "compatible_versions": [
+        "v1"
+      ],
+      "cosmos_sdk_version": "0.46.7",
+      "cosmwasm_enabled": true,
+      "genesis": {
+        "genesis_url": "https://github.com/Bluechip23/bluechip/blob/main/genesis.json"
+      },
+      "versions": [
+        {
+          "name": "v1",
+          "recommended_version": "v1",
+          "compatible_versions": [
+            "v1"
+          ],
+          "cosmos_sdk_version": "0.46.7",
+          "cosmwasm_enabled": true,
+        }
+      ]
+    },
+    "peers": {
+      "seeds": [],
+      "persistent_peers": [
+        {
+          "id": "3x4pynlp86prhcmtns742kgsgu7pjtzjpk8d93",
+          "address": "http://67.28.8.88:26659/",
+          "provider": "CryptoDungeon"
+        }
+      ]
+    },
+    "images": [
+      {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluechip/images/bluechip.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/bluechip/images/bluechip.svg"
```

Comment:
I think the svg might be invalid

Reply:
Please remove the svg. It looks low quality and unlike the png.

Reply:
The svg is not scalable vectors, it's just an embedded image. Please remove unless there is an actual scalable vector file.

Reply:
I have deleted it. I will have to find a resource to help make an svg file. Is this something required for initial submittal?

Reply:
I think the last thing is to remove iamges[0].svg from this chain file and from the assetlist.